template_config.yml

```diff
@@ -22,13 +22,15 @@ plugin_dash: pulp-rpm
 plugin_dash_short: rpm
 plugin_name: pulp_rpm
 plugin_snake: pulp_rpm
+publish_docs_to_pulpprojectdotorg: false
 pulp_settings:
   allowed_export_paths:
   - /tmp
   allowed_import_paths:
   - /tmp
 pulpcore_branch: master
 pulpcore_pip_version_specifier: null
+pulpprojectdotorg_key_id: null
```

Comment:
@dkliban where can we get this?

Reply:
@fao89 , we won't publish docs to pulpproject.org with this release